samples/nrf9160/lwm2m_client/overlay-assist-pgps.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_NEWLIB_LIBC_FLOAT_PRINTF=y
+CONFIG_LWM2M_CLIENT_UTILS_GNSS_ASSIST_OBJ_SUPPORT=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_OBJ_SUPPORT=n
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_EVENTS=y
+CONFIG_NRF_CLOUD_PGPS=y
+CONFIG_NRF_CLOUD_PGPS_TRANSPORT_NONE=y
+CONFIG_LWM2M_CLIENT_UTILS_LOCATION_ASSIST_PGPS=y
+#Enable LwM2M 1.1
+CONFIG_LWM2M_VERSION_1_1=y
+CONFIG_LWM2M_COMPOSITE_PATH_LIST_SIZE=10
+#Enable SenML-CBOR content format
+CONFIG_LWM2M_RW_SENML_CBOR_SUPPORT=y
+CONFIG_ZCBOR=y
+CONFIG_ZCBOR_CANONICAL=y
+CONFIG_LWM2M_RW_SENML_CBOR_RECORDS=35
+#Enable SenML-JSON content format
+CONFIG_BASE64=y
```

Comment:
Are we need senml-json support?

Reply:
Good point, the base was copied from old configs, probably we don't need it.

Reply:
Done